include/tallies/CellTally.h

```diff
@@ -20,9 +20,13 @@
 
 #include "TallyBase.h"
 #include "OpenMCCellAverageProblem.h"
+#include "MooseMesh.h"
 
 #include "openmc/tallies/filter_cell.h"
 
+class OpenMCCellAverageProblem;
+class MooseMesh;
```

Comment:
why are these here? when you `#include` the header files, you don't need these. These are forward declarations, which may be something used when you DON'T include a header file. Remove